dozer-spring-boot-auto-configuration/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2005-2017 Dozer Project
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>com.github.dozermapper</groupId>
+        <artifactId>dozer-plugins-parent</artifactId>
+        <version>6.2.0-SNAPSHOT</version>
+        <relativePath>../plugins-parent</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>dozer-spring-boot-auto-configuration</artifactId>
+    <packaging>pom</packaging>
+    <name>Spring Boot AutoConfiguration :: Dozer</name>
+
+    <dependencyManagement>
+        <dependencies> <!-- The parent should provide all that -->
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-dependencies</artifactId>
+                <version>1.5.8.RELEASE</version>
```

Comment:
Probably it makes sense to extract this version to a property in `bom-dependencies/pom.xml`

Reply:
It did not not work. It will work only in case if `bom-dependencies` is a direct parent of `dozer-spring-boot-auto-configuration`.

Reply:
Sorry for misleading comment, it's not only about the property, but also about the dependency itself. 
I.e. if we define in `bom-dependencies/pom.xml`:
```xml
        <dependencyManagement>
        ...
            <dependency>
                <groupId>org.springframework.boot</groupId>
                <artifactId>spring-boot-dependencies</artifactId>
                <version>${spring-boot.version}</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
```
And then in `dozer-spring-boot-auto-configuration/pom.xml` define just:
```xml
            <dependency>
                <groupId>org.springframework.boot</groupId>
                <artifactId>spring-boot-dependencies</artifactId>
            </dependency>
```
It should work fine.

Reply:
Yes. That make sense.